requirements.txt

```diff
@@ -4,5 +4,5 @@ more_executors>=2.11.4
 pushcollector>=1.3.0
 pushsource>=2.50.0
 strenum>=0.4.15
-starmap-client>=2.3.0
-cloudpub>=1.4.0
+starmap-client>=2.4.0
```

Comment:
I though name of package was changed to starmap_client

Reply:
It's more like a normalization for PEP625 but it works with either `-` or `_` on requirements